legal-api/report-templates/correction.html

```diff
@@ -0,0 +1,55 @@
+[[macros.html]]
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <title>Register Correction Application</title>
```

Comment:
do we want to remove the word "Register" or use a different title altogether?

Reply:
Agreed, I think it should wither be _Registry Correction_ or just _Correction_

Reply:
I will verify this with Mihai

Reply:
Confirmed with Mihai. This is what business want, we have the same title in Firm Correction.